src/generic_ui/polymer/cloud-install.html

```diff
@@ -32,20 +29,26 @@
     }
     uproxy-button {
       background-color: #34AFCE;  /* blue */
+      margin: 4px 0;
       padding: 0.5em;
       width: 100%;
       font-size: 14px;
+      text-transform: none;
```

Comment:
Ok, as of b1fdfec we are no longer using `text-transform: uppercase` on uproxy-buttons and various other places.

![screen shot 2016-06-23 at 15 38 06](https://cloud.githubusercontent.com/assets/64992/16317339/a3ea6006-3958-11e6-91c0-20402b7be525.png)
![screen shot 2016-06-23 at 15 37 58](https://cloud.githubusercontent.com/assets/64992/16317342/a6aaf38c-3958-11e6-9378-02fcf3c72067.png)